orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/tasks/PageApplicationOwnerTask.groovy

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.netflix.spinnaker.orca.echo.tasks
+
+import com.netflix.spinnaker.orca.DefaultTaskResult
+import com.netflix.spinnaker.orca.RetryableTask
+import com.netflix.spinnaker.orca.TaskResult
+import com.netflix.spinnaker.orca.echo.EchoService
+import com.netflix.spinnaker.orca.pipeline.model.Stage
+import groovy.transform.CompileStatic
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+import java.util.concurrent.TimeUnit
+
+import static com.netflix.spinnaker.orca.ExecutionStatus.SUCCEEDED
+
+@Component
+@CompileStatic
+class PageApplicationOwnerTask implements RetryableTask {
+  long backoffPeriod = 15000
+  long timeout = TimeUnit.MINUTES.toMillis(5)
+
+  @Autowired
+  EchoService echoService
+
+  @Override
+  TaskResult execute(Stage stage) {
+    echoService.create(
```

Comment:
I initially had it here ... but opt'd for the generic bit being that we want to page "by application" and orca not having to actually know whether it's pager duty, victor ops, etc.

From echo's perspective, I suspect there could be multiple notification services that can handle an `INCIDENT` (but expect only one would be enabled).